test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -3180,26 +3136,6 @@ static public void testValueClassHashCode() throws Throwable {
 		assertNotEquals(h1, h5);
 	}
 
-	@Test(priority=1)
-	static public void testPrimitiveClassHashCode() throws Throwable {
```

Comment:
Is there a reason why this case is removed ?

Reply:
This to match what was removed in the lw5 test suite https://github.com/eclipse-openj9/openj9/pull/18505#issue-2004727285

Reply:
I see. There is `testValueClassHashCode()` above this.